robottelo/ui/base.py

```diff
@@ -831,3 +837,31 @@ def perform_entity_action(self, action_name):
         """
         self.click(common_locators['kt_select_action_dropdown'])
         self.click(common_locators['select_action'] % action_name)
+
+    def _configure_orgs(self, orgs, org_select):
```

Comment:
Hm. That is really bad idea to put these methods to the most base class of framework. Move them to `robottelo/ui/ldapauthsource.py` or wherever you need them